src/garage/torch/algos/vpg.py

```diff
@@ -308,5 +361,12 @@ def _log(self, itr, paths):
         tabular.record('StdReturn', np.std(undiscounted_returns))
         tabular.record('MaxReturn', np.max(undiscounted_returns))
         tabular.record('MinReturn', np.min(undiscounted_returns))
+        tabular.record('{0}/LossBefore'.format(self.policy.name), loss_before)
```

Comment:
you can use `tabular.prefix` for this: https://github.com/rlworkgroup/dowel/blob/master/src/dowel/tabular_input.py

you might also be interested in `tabular.record_misc_stat`